src/webgpu/shader/execution/expression/call/builtin/subgroupBallot.spec.ts

```diff
@@ -0,0 +1,202 @@
+export const description = `
+Execution tests for subgroupBallot
+
+Note: There is a lack of portability for non-uniform execution so these tests
+restrict themselves to uniform control flow or returning early.
+Note: There is no guaranteed mapping between subgroup_invocation_id and
+local_invocation_index. Tests should avoid assuming there is.
+
+Note: This currently uses a parameter-less version of ballot.
```

Comment:
This isn't described in the proposal doc.
https://github.com/gpuweb/gpuweb/blob/main/proposals/subgroups.md

Reply:
Agreed, a tint issue. Filed https://issues.chromium.org/issues/353736338